package.json

```diff
@@ -8,8 +8,8 @@
     "lint": "eslint .",
     "lint-fix": "eslint . --fix",
     "test-config": "node scripts/local-test.js",
-    "test-integration": "./scripts/tests/testIntegration.sh",
-    "test-unit": "./scripts/tests/testUnit.sh",
+    "test-integration": "sh scripts/tests/testIntegration.sh",
+    "test-unit": "sh scripts/tests/testUnit.sh",
```

Comment:
Looks good to me. Works on Git Bash on Windows and Linux as well.
Thanks for the changes!

Reply:
Right! Thank you. 👍